model.py

```diff
@@ -2239,13 +2239,9 @@ class Edition(Base):
     primary_identifier_id = Column(
         Integer, ForeignKey('identifiers.id'), index=True)
 
-    # A Edition may be associated with a single Work.
-    work_id = Column(Integer, ForeignKey('works.id'), index=True)
+    # An Edition may be the presentation edition for a single Work.
+    work = relationship("Work", uselist=False, backref="presentation_edition")
```

Comment:
I'm not sure what to call this. For LicensePool, it's is_presentation_for, but that's confusing now that it can also be the presentation for a Work.